src/natupnp_v1.erl

```diff
@@ -169,9 +170,32 @@ add_port_mapping1(#nat_upnp{ip=Ip, service_url=Url}, Protocol, InternalPort,
             Now = nat_lib:timestamp(),
             MappingLifetime = Lifetime - (Now - Start),
             {ok, Now, InternalPort, ExternalPort, MappingLifetime};
```

Comment:
what should return there if PermanentLease have been set? would returning `permanent_lease` or `infinity` be ok?

Reply:
yes, both seems fine 👍

Reply:
it will return infinity for permanent leases. Thanks for the feedback !